python/src/iceberg/io/base.py

```diff
@@ -44,7 +44,7 @@ class InputStream(Protocol):  # pragma: no cover
     def read(self, size: int) -> bytes:
         ...
 
-    def seek(self, offset: int, whence: int) -> None:
```

Comment:
This is now also in line with BinaryIO 👍🏻

Reply:
What does `whence` do?

Reply:
I see below.